app/modules/check_storage.js

```diff
@@ -2,13 +2,19 @@ function checkStorage() {
   let tasks;
   let counter;
   const tested = [];
-  if (localStorage.getItem('tasks') === null) {
+  if (
+    localStorage.getItem('tasks') === null
+    || localStorage.getItem('tasks') === '[]'
+  ) {
     tasks = [];
-    counter = 0;
+    counter = tasks.length + 1;
   } else {
     tasks = JSON.parse(localStorage.getItem('tasks'));
     counter = tasks[tasks.length - 1].id + 1;
+    // counter = setTimeout(() => tasks.length, 1000);
+    // console.log(counter);
```

Comment:
- [ ][OPTIONAL] it would be better if you remove the uncommented code in order to keep your file clean and pro